python/lsst/ts/ess/common/constants.py

```diff
@@ -37,6 +40,11 @@
 # The value emitted by a disconnected channel of the temperature sensors.
 DISCONNECTED_VALUE = "9999.9990"
 
+"""Telemetry prefixes for the LD-250 sensor."""
+NOISE_PREFIX = "WIMLN"
+STATUS_PREFIX = "WIMST"
+STRIKE_PREFIX = "WIMLI"
```

Comment:
I strongly suggest including the sensor name (or a hint to it) in the constant name, to make it relevant what domain these constants apply to. For readability consider creating a class:
```
class LD250TelemetryPrefix:
    NOISE = "WIMLN"
    ...
```
It could be an enum class if you prefer; what would support iteration and checking for membership, in case that would be useful.